Eixozmoviment2/Eixozmovimento2/Eixozmovimento2.pde

```diff
@@ -1,64 +1,76 @@
 float x, y, z;
+float screenW = 600;
+int screenH = 600;
+float a = 0.0;
+float degree = TWO_PI/180.0;
+float cubesPerLevel = 5;
+float angle = 360/5.0;
+int radius = 200;
```

Comment:
isto e raio do circulo imaginario, para formar o anel de cubos

Reply:
`float a` - podes apagar, foi algo que me esqueci nao esta ai a fazer nada.

`float degree` - `float` diz que é a propriedade é um valor decimal, `degree` vai ser o numero em radianos que vai ser usado para fazer o calculo na posicao do circulo.

`float angle ` -  `angle ` vai calcular o angulo entre cada cubo, neste caso estamos a dividir 360 graus ou circulo, pelo o numero de cubos que queremos no circulo

Reply:
sabes algo de trigonometria?
este tem uma explicacao
https://math.stackexchange.com/questions/260096/find-the-coordinates-of-a-point-on-a-circle

Mas podes procurar outras definicoes